package.json

```diff
@@ -88,6 +88,7 @@
     "ember-compatibility-helpers": "^1.2.5",
     "ember-disable-prototype-extensions": "^1.1.2",
     "ember-export-application-global": "^2.0.1",
+    "ember-fetch": "^8.1.1",
```

Comment:
what does this have to do with IE testing?

Reply:
Need it for https://github.com/hjdivad/ember-m3/blob/master/tests/dummy/app/adapters/-json-api.js#L5